R/zzz.R

```diff
@@ -7,7 +7,13 @@
   if (has_ggplot2_3.0)
     s3_register("ggplot2::autoplot", "sfnetwork")
 
-  s3_register("spatstat::as.linnet", "sfnetwork")
+  has_spatstat_2.0 =
+    requireNamespace("spatstat", quietly = TRUE) &&
+    utils::packageVersion("spatstat") >= "2.0.0"
+
+  if (has_spatstat_2.0) {
+    s3_register("spatstat.linnet::as.linnet", "sfnetwork")
```

Comment:
In think here we could use:

```
  if (has_spatstat_2.0) {
    suppressPackageStartupMessages(
      s3_register("spatstat.linnet::as.linnet", "sfnetwork")
    )
  }
```

such that now spatstat related startup messages are printed

Reply:
Fixed with the last commits!